Parser/SBE19Parse.m

```diff
@@ -412,7 +412,7 @@
 headerExpr   = '^\*\s*(SBE \S+|SeacatPlus)\s+V\s+(\S+)\s+SERIAL NO.\s+(\d+)';
 %BDM (18/2/2011) - new header expressions to reflect newer SBE header info
 headerExpr2  = '<HardwareData DeviceType=''(\S+)'' SerialNumber=''(\S+)''>';
-headerExpr3  = 'Sea-Bird (\S+) Data File:';
+headerExpr3  = ''(?:Sea-Bird)(.*)(?:Data File\:)'';
```

Comment:
Having 2 single quotes in a row produces an error. Will try replacing that with just:
```'(?:Sea-Bird)(.*)(?:Data File\:)'```